.circleci/config.yml

```diff
@@ -21,6 +20,7 @@ workflows:
           platform: amd64
           executor: greenzie/debianize_amd64
           clone_submodules: true
+          
```

Comment:
```suggestion
```
Was this newline intentional? It looks like we use a single newline between other similar blocks in this file.